omod/src/main/resources/config.xml

```diff
@@ -128,5 +128,10 @@
 		<description>Set provider attribute type uuid</description>
 	</globalProperty>
 
-</module>
+	<globalProperty>
+		<property>${project.parent.artifactId}.narrativesOverridePropertyFile</property>
+		<defaultValue></defaultValue>
+		<description>Absolute path of narrative override properties file (prefixed with "file:")</description>
```

Comment:
Is this requiring that it be prefixed with `file:`? I would prefer to do something like:

Does it start with `classpath:`? Use the `classpath`. Otherwise if it doesn't start with `file:` add file. We should also be careful when loading a file from a user-specified directory. We should check that it's in the OpenMRS Application Data directory, ends in `.properties`, etc.

Reply:
Yeah, actually it does support the `classpath:` and the `file:` prefix, since even this uses the `NarrativeTemplateManifest.loadResource()` method. I'll add it in the description too.

Okay, I'll add some validation for `file:` prefixed paths while loading the file.

Reply:
@ibacher Does the properties file have to necessarily be in the OpenMRS Application Data directory?

Reply:
I suppose that's less of a concern really. We do want make sure that users can supply something like `/opt/OpenMRS/overrides/mynarratives.properties`, however, without needing to prefix it with `file:`.

Reply:
Oh okay. But then we'll be needing to append the `file:` prefix since it also uses `NarrativeTemplateManifest.loadResource()` method to load the properties file, which further needs the `file:` or `classpath:` prefix. (Later when we add support for `openmrs:` we would be able to use it for loading properties file too)

Reply:
@ibacher Also, while validating the narrative override properties filepath, in case of some error, should we raise a ValidationError or just log the error and continue without overriding the properties?

Reply:
> We should also be careful when loading a file from a user-specified directory. We should check that it's in the OpenMRS Application Data directory, ends in .properties, etc.

@ibacher I was thinking, do we actually need to validate it on our end? Because, validation for existence of the file is already being done in the [`NarrativeTemplateManifest.loadResource()`](https://github.com/jamesagnew/hapi-fhir/blob/29bf6b8171f48f14ec2c44c997122c3917444942/hapi-fhir-base/src/main/java/ca/uhn/fhir/narrative2/NarrativeTemplateManifest.java#L184) method. We could add a validation for `.properties` extension if is needed though.